stdlib/websub/src/main/ballerina/src/websub/subscriber_client.bal

```diff
@@ -52,11 +55,13 @@ public type SubscriptionClient client object {
                                   redirectCount);
     }
 
-    # Sends an unsubscription request to a WebSub Hub.
-    #
-    # + unsubscriptionRequest - The `SubscriptionChangeRequest` containing unsubscription details
-    # + return - `SubscriptionChangeResponse` indicating unsubscription details, if the request was successful else
-    #            `error` if an error occurred with the unsubscription request
+# Sends an unsubscription request to a WebSub Hub.
+# ```ballerina
+# websub:SubscriptionChangeResponse|websub:error response = websubHubClientEP->unsubscribe(subscriptionRequest);
+# ```
+# + unsubscriptionRequest - The `SubscriptionChangeRequest` containing unsubscription details
+# + return - An unsubscription details if the request was successful or else an `error` if an error occurred
+#            with the unsubscription request
```

Comment:
```suggestion
#            with the unsubscription request
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.